docs/rbe/rbe.md

```diff
@@ -0,0 +1,161 @@
+# RBE for Flutter Engine Developers
+
+## Overview
+
+This is documentation on setting up RBE for building the Flutter engine. It is
+expected to work for Googlers only on corp Linux, macOS, and Windows hosts,
+including cloudtop instances.
+
+## Getting started
+
+The first step is to add an entry to the `.gclient` file. The entry to add is
+`"use_rbe": True` in the `custom_vars` section. It should look like this:
+
+After making this edit, you must be authenticated as a Googler by CIPD so that
+the RBE configurations can be downloaded from the `flutter_internal`
+[CIPD bucket](https://chrome-infra-packages.appspot.com/p/flutter_internal):
+
+After authentication successfully, run `gclient sync -D`.
+
+## Running an RBE build
+
+In the engine repo, all RBE builds must be initiated through the `et` tool whose
+entrypoint is the script `//flutter/bin/et`. This is so that the local RBE
+proxy is correctly initialized and shut down around invocations of `ninja`.
+
+### gcloud
+
+Before running an RBE build, you must be authenticated with the Google cloud
+project that owns the RBE worker pool. A gcloud SDK is checked out with the
+engine repo under e.g. `//flutter/buildtools/mac-arm64/gcloud`.
+(Replace `mac-arm64` with the directory name that is correct for your host
+platform.)
+
+The `gcloud` tool in this SDK must be on your path. The tool lives under
+`//flutter/buildtools/mac-arm64/gcloud/bin`, which is the path to add to your
+`PATH` environment variable. Alternatively, you can get the gcloud SDK on your
+path by installing it on your system by following the instructions at
+[https://cloud.google.com/sdk/docs/install]
+(https://cloud.google.com/sdk/docs/install).
+
+On macOS, before running the `gcloud` command ensure that `python3` is on your
+path, and does not come from e.g. homebrew. The command `which python3` should
+return `/usr/bin/python3`.
```

Comment:
I think more documentation would be helpful here, after talking to @johnmccutchan what I ended up doing was `gcloud init` followed by selecting my `@google` account, followed by entering a project id that John shared with me (Not putting here because I don't know if its a problem to list it externally, but if not we should probably also add that here?)

Hopefully that is correct?

Reply:
Thanks for catching this. I messed up my initial import of the markdown, but I think I have everything corrected except for the links in https://github.com/flutter/engine/pull/52442.